build_runner/lib/src/watcher/asset_change.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:build/build.dart';
+import 'package:path/path.dart' as path;
+import 'package:watcher/watcher.dart';
+
+import '../package_graph/package_graph.dart';
+
+/// Represents an [id] that was modified on disk as a result of [type].
+class AssetChange {
+  /// Asset that was changed.
+  final AssetId id;
+
+  /// What caused the asset to be detected as changed.
+  final ChangeType type;
+
+  /// Create a new change record.
```

Comment:
[nit] I think some of these comments are superfluous.

Reply:
Removed this one at least.